graphrag_sdk/ontology.py

```diff
@@ -152,6 +154,29 @@ def from_kg_graph(graph: Graph, sample_size: int = 100,):
         
         return ontology
     
+    @staticmethod
+    def process_attributes_from_graph(attributes: list[list[list[str]]]) -> list[Attribute]:
```

Comment:
Maybe instead of a class method which is exposed to the public, have this as a file function which isn't exported.
also it would be nicer if we can pass something less complex than `list[list[list[str]]]`